src/web-auth/transaction-manager.js

```diff
@@ -45,17 +47,19 @@ TransactionManager.prototype.generateTransaction = function(
 ) {
   state = state || random.randomString(this.keyLength);
   nonce = nonce || (generateNonce ? random.randomString(this.keyLength) : null);
-
-  this.storage.setItem(
-    this.namespace + state,
-    {
-      nonce: nonce,
-      appState: appState,
-      state: state,
-      lastUsedConnection: lastUsedConnection
-    },
-    { expires: times.MINUTES_30 }
-  );
+  var isHostedLoginPage = windowHelper.getWindow().location.host === this.options.domain;
+  if (!isHostedLoginPage) {
```

Comment:
It would be more clear to just check the condition rather than set a var then negate it.

Reply:
I like the var because it shows what I'm trying to do